1-js/02-first-steps/15-function-expressions/article.md

```diff
@@ -1,84 +1,84 @@
-# Function expressions
+# Wyrażenia funkcyjne
 
-In JavaScript, a function is not a "magical language structure", but a special kind of value.
+W JavaScripcie, funkcje nie są "magicznymi konstrukcjami językowymi", lecz po prostu rodzajem wartości.
 
-The syntax that we used before is called a *Function Declaration*:
+W poprzednich przykładach, używaliśmy składni nazywanej *deklaracją funkcji*:
 
 ```js
 function sayHi() {
-  alert( "Hello" );
+  alert( "Cześć" );
 }
 ```
 
-There is another syntax for creating a function that is called a *Function Expression*.
+Innym sposobem na utworzenie funkcji jest tzw. *wyrażenie funkcyjne*.
 
-It looks like this:
+Wygląda ono następująco:
 
 ```js
 let sayHi = function() {
-  alert( "Hello" );
+  alert( "Cześć" );
 };
 ```
 
-Here, the function is created and assigned to the variable explicitly, like any other value. No matter how the function is defined, it's just a value stored in the variable `sayHi`.
+Funkcja jest tworzona i przypisywana do zmiennej - tak, jakby była to zwykła wartość. Nie ważne, w jaki sposób zostanie ona zdefiniowana - będzie to po prostu wartość, przechowywana w zmiennej `sayHi`.
```

Comment:
You skipped the word `explicitly`. Pleace change: `Funkcja jest tworzona i przypisywana do zmiennej` to `Funkcja jest tworzona i jawnie przypisywana do zmiennej`.